stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Config.kt

```diff
@@ -4,39 +4,109 @@ import com.google.gson.annotations.SerializedName
 import java.util.Date
 
 public data class Config(
+
+    /**
+     * Date of channel creation.
+     */
     @SerializedName("created_at")
-    var created_at: Date? = null,
+    val created_at: Date? = null,
 
+    /**
+     * Date of last channel update.
+     */
     @SerializedName("updated_at")
-    var updated_at: Date? = null,
+    val updated_at: Date? = null,
+
+    /**
+     * The name of the channel type must be unique per application
+     */
+    val name: String = "",
 
+    /**
+     * Controls if typing indicators are shown. Enabled by default.
+     */
     @SerializedName("typing_events")
     var isTypingEvents: Boolean = false,
 
+    /**
+     * Controls whether the chat shows how far you’ve read. Enabled by default.
+     */
     @SerializedName("read_events")
     var isReadEvents: Boolean = false,
 
+    /**
+     * Determines if events are fired for connecting and disconnecting to a chat. Enabled by default.
+     */
     @SerializedName("connect_events")
-    var isConnectEvents: Boolean = false,
+    val isConnectEvents: Boolean = false,
 
+    /**
+     * Controls if messages should be searchable (this is a premium feature). Disabled by default.
+     */
     @SerializedName("search")
-    var isSearch: Boolean = false,
+    val isSearch: Boolean = false,
 
+    /**
+     * If users are allowed to add reactions to messages. Enabled by default.
+     */
     @SerializedName("reactions")
-    var isReactionsEnabled: Boolean = false,
+    val isReactionsEnabled: Boolean = false,
 
+    /**
+     * Enables message threads and replies. Enabled by default.
+     */
     @SerializedName("replies")
-    var isRepliesEnabled: Boolean = false,
+    val isRepliesEnabled: Boolean = false,
 
+    /**
+     * Determines if users are able to mute other users. Enabled by default.
+     */
     @SerializedName("mutes")
-    var isMutes: Boolean = false,
+    val isMutes: Boolean = false,
+
+    /**
+     * Allows image and file uploads within messages. Enabled by default.
+     */
+    @SerializedName("uploads")
+    val isUploads: Boolean = false,
+
+    /**
+     * Determines if URL enrichment enabled to show they as attachments. Enabled by default.
+     */
+    @SerializedName("url_enrichment")
+    val isUrlEnrichment: Boolean = false,
+
+    @SerializedName("custom_events")
+    val isCustomEvents: Boolean = false,
 
+    @SerializedName("push_notifications")
+    val isPushNotifications: Boolean = false,
```

Comment:
It's good to mention in `changelog`, that now we support such fields in our offline module